module/GeebyDeeby/view/layout/layout.phtml

```diff
@@ -1,23 +1,21 @@
-<? echo $this->doctype(); ?>
+<?=$this->doctype(); ?>
```

Comment:
You seem to have changed `<? echo` to `<?=` throughout this file. Not sure why that is part of this pull request. If we want to make this change (which probably makes sense) it would be better to make that change in master -- it's not specific to the dimenovels.org branch.